common/src/main/kotlin/com/sourceclear/agile/piplanning/objects/Things.kt

```diff
@@ -121,7 +124,7 @@ data class StoryRequestI(
     val storyPoints: Int,
     val storyEpicId: Long,
     val storySprintId: Long,
-    val notes: String
+    val notes: String = ""
```

Comment:
frontend sends null here if notes were not touched.

Reply:
Hmm, should the front end always fill this in?